download/upgradeRecipe/upgradeRecipe8.adoc

```diff
@@ -50,4 +50,176 @@ Any backwards incompatible changes are annotated with a [.label.label-danger.lab
 Due to the limited usage of OSGi and the maintenance burden it brings,
 the OptaPlanner jars in the 8.x series no longer include OSGi metadata in their `META-INF/MANIFEST.MF` file.
 
+[.upgrade-recipe-major]
+=== `filterClassList` replaced by a single filterClass
 
+This change applies to a configuration of `EntitySelector`, `ValueSelector` and `MoveSelector` in both
```

Comment:
Use _active tense_:
So instead of "This change applies to ...", something like "... now has ... " or "... is now ..." .

For example "The configuration of`EntitySelector`, `ValueSelector` and `MoveSelector` now has a single `filterclass`, in both ..."